packages/buttons/src/types/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { AnchorHTMLAttributes, ButtonHTMLAttributes, HTMLAttributes, SVGAttributes } from 'react';
+
+export const SIZE = ['small', 'medium', 'large'] as const;
```

Comment:
It looks like this is `undefined` at runtime and React throws a `factoryWithTypeCheckers.jss:23 Warning: Invalid argument supplied to oneOf, expected an array` in development mode.

Reply:
I think I may have had stale `node_modules` earlier, but I'm no longer seeing this warning.